src/App.js

```diff
@@ -9,40 +9,41 @@ import Input from './components/Input';
 import Button from './components/Button';
 import List from './components/List';
 import Counter from './components/Counter';
+import ErrorMessage from './components/ErrorMessage';
 import { addTask, removeTask } from './store/actions/listActions';
 
 class App extends Component {
   state = {
     value: '',
-    isInputEmpty: false,
-    isValueExist: false,
+    isError: false,
```

Comment:
according to english, I would name it `hasError`, because input value can have error rather than being an error.